tailwind.config.js

```diff
@@ -6,6 +6,9 @@ export default {
       'lato': ['Lato'],
       'oswald': ['Oswald'],
     },
+    fontSize: {
+      nav: ['24px', '28.8px'],
```

Comment:
Even though this matches the Figma design, it looks too large. I would change this to 16px, and you don't really need to set the line height (the second value).